src/routes/safe/components/Transactions/TxList/TxDetails.tsx

```diff
@@ -95,6 +95,7 @@ export const TxDetails = ({ transaction }: TxDetailsProps): ReactElement => {
   const currentUser = useSelector(userAccountSelector)
   const hasModule = transaction.txDetails && isModuleExecutionInfo(transaction.txDetails.detailedExecutionInfo)
   const isMultiSend = data && isMultiSendTxInfo(data.txInfo)
+  const noOwners = !!(data && data.txInfo.type === 'Transfer' && data.txInfo.direction === 'INCOMING')
```

Comment:
Don't we have some enums for those types?

Reply:
We have helper functions for check at least one of these.

Reply:
All tests were passing after this change. However, this logic was modified.